app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -330,6 +340,10 @@ const SendBitcoinConfirmationScreen: React.FC<Props> = ({ route }) => {
     DisplayCurrency,
   )
 
+  if (paymentDetail.settlementAmount.amount < 546) {
```

Comment:
@dolcalmi @vindard dust threshold is a setting on the galoy.yaml https://github.com/GaloyMoney/galoy/blob/main/galoy.yaml#L168. should we expose that to make the mobile app more dyanmic on what threshold it should accept for sending onchain transactions?